src/pages/blog/fixed-cost-is-hurting-your-business.pug

```diff
@@ -48,8 +48,8 @@ block pageComponents
 
             p.blog-post__text When costed ahead of time, you simply cannot plan for the unknown. As new ideas get discussed, or features emerge, who decides if it’s actually in scope or not?
 
-            blockquote
-              q
+            blockquote.blog-blockquote
+              q.main-blockquote
```

Comment:
From a 30 second Google, this isn't how yo use the q element (I realise it was there before you got to it). It looks like a `p` element is standard inside a `blockquote`, and that the `q` is used for 'inline quotes'.